pythermalcomfort/models/two_nodes_gagge_ji.py

```
@@ -0,0 +1,455 @@
+        wettedness_dif = 1 / (2 + 2.46 * he_n)
+        wp = (
+            wettedness_dif + (1 - wettedness_dif) * p_rsw
+        )  # skin wetness (between 0.06 and 1)
+
+        # Calculate skin wettedness, by taking the ISO PHS evaporation efficiency formula: eff = 1 - 0.5 * w**2
+        w = (math.sqrt(2 * wp**2 + 1) - 1) / wp
+
+        # Limit w
+        if w > w_max:
+            # print("Skin wetness exceeds max value ${:.2f}".format(w_max))
+            w = w_max
```

Pylint (3.3.7)

[refactor] Consider using 'w = min(w, w_max)' instead of unnecessary if block

(R1730)